src/assets/scss/app.scss

```
@@ -0,0 +1,19 @@
+@import "~bulma";
+@import "~bulma-timeline";
+
+.navbar {
+  border-bottom: 1px solid #eee;
+  margin-bottom: 8px;
+}
+
+.gh-username img{
+  width: 32px;
+}
+
+#profile-bio{
+  margin-top:10px;
+}
+
+.timeline .timeline-item {
```

It's SASS, so it can be like that:
```suggestion
.timeline {
  &-item {
    padding-bottom: 1em;
  }
}
```